pkg/transforms/policy.go

```diff
@@ -37,6 +41,121 @@ func PolicyResourceBuilder(p *p.Policy) *PolicyResource {
 	if okns && okpp {
 		node.Properties["_parentPolicy"] = pnamespace + "/" + ppolicy
 	}
+
+	return &PolicyResource{node: node}
+}
+
+// For cert, config, operator policies.
```

Comment:
I think this comment needs to be updated based on your latest changes.

Reply:
getPolicyCommonProperties Also used in Operator too

Reply:
Right, but the code isn't collecting all of the managed fields anymore and this documentation says it does.

Reply:
let me change the comment